backend/src/zac/api/permissions.py

```diff
@@ -123,15 +129,15 @@ def has_permission(self, request: Request, view: APIView) -> bool:
         if request.user.is_superuser:
             return True
 
-        # first check  if the user has permissions for any object
-        if super().has_permission(request, view) is False:
-            return False
-
         serializer = view.get_serializer(data=request.data)
         # if the serializer is not valid, we want to see validation errors -> permission is granted
         if not serializer.is_valid():
             return True
 
+        # first check  if the user has permissions for any object
+        if super().has_permission(request, view) is False:
+            return False
+
```

Comment:
I think PEP 8 states this should probably be rewritten as:

```suggestion
        if not super().has_permission(request, view):
            return False

```